migrations/versions/0363_cancelled_by_api_key.py

```diff
@@ -0,0 +1,62 @@
+"""
+
+Revision ID: 0363_cancelled_by_api_key
+Revises: 0362_broadcast_msg_event
+Create Date: 2022-01-25 18:05:27.750234
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+revision = '0363_cancelled_by_api_key'
+down_revision = '0362_broadcast_msg_event'
+
+
+def upgrade():
```

Comment:
What are we doing with the data in the production broadcast_message table?
There currently are broadcast messages, unless we are purging that data we will need to do this migration in a different way. Let me know if you want my assistance with that migration.